ddocs/medic-schema/schema/app_settings

```diff
@@ -0,0 +1,1223 @@
+{
```

Comment:
Used a variety of project settings as a base to make it. This is another reason I'm for keeping the schema in couch, although the rules are loose right now, there's still a chance a configurer could make an unforeseen declaration that would not be schema compliant. In such a case, accessibility would make their life easier. What are your thoughts on this?

I'll check out the old one and see what can be lifted.

Reply:
I'm going to tag @garethbowen in here because I think that's a question for his level.

However, while I'm here my thoughts are this are:
 - API and the settings schema should have a 1:1 relationship, and shouldn't be able to be bypassed, circumvented or updated by users, and we should base our architecture off that. Because of that I think the settings file might as well be on disk in API.
 - If we find that the schema is wrong in some way that would be a product bug like any other, and we'd issue a patch
 - Worst case there is always the escape hatch of just updating the `settings` document directly

Reply:
Totally agree with you Stefan, making the schema easily accessible will encourage the bad practice of casual updates which won't make it to the main release.

Reply:
I've moved schema to API

Reply:
Fields from the previous schema have been updated on the new one